deeppavlov/metrics/accuracy.py

```diff
@@ -109,3 +109,21 @@ def per_item_dialog_accuracy(y_true, y_predicted):
     examples_len = len(y_true)
     correct = sum([y1.strip().lower() == y2.strip().lower() for y1, y2 in zip(y_true, y_predicted)])
     return correct / examples_len if examples_len else 0
+
+
+@register_metric('acc')
+def round_accuracy(y_true, y_predicted):
+    """
+    Rounds predictions and calculates accuracy in terms of absolute coincidence.
+
+    Args:
+        y_true: list of true values
+        y_predicted: list of predicted values
+
+    Returns:
+        portion of absolutely coincidental samples
+    """
+    predictions = list(map(lambda x: round(x), y_predicted))
```

Comment:
```python
list(map(round, y_predicted))
```
or
```python
[round(x) for x in y_predicted]
```
round is already a function, no need to create a lambda from it

Reply:
or you can skip this line and just do
```python
correct = sum([y1 == round(y2) for y1, y2 in zip(y_true, y_predicted)])
```

Reply:
corrected